docs/en/user_guides/1_config.md

```diff
@@ -0,0 +1,120 @@
+
+We incorporate modular and inheritance design into our config system,
+which is convenient to conduct various experiments.
+If you wish to inspect the config file,
+you may run `python tools/misc/print_config.py /PATH/TO/CONFIG` to see the complete config.
+
+## A brief description of a complete config
+
+A complete config usually contains the following primary fields:
+
+- `model`: the basic config of model, which may contain `data_preprocessor`, modules (e.g., `detector`, `motion`),`train_cfg`, `test_cfg`, etc.
+- `train_dataloader`: the config of training dataloader, which usually contains `batch_size`, `num_workers`, `sampler`, `dataset`, etc.
+- `val_dataloader`: the config of validation dataloader, which is similar with `train_dataloader`.
+- `test_dataloader`: the config of testing dataloader, which is similar with `train_dataloader`.
+- `val_evaluator`: the config of validation evaluator. For example, `type='CocoVideoMetric'` for VID task on the ILSVRC benchmark, `type='MOTChallengeMetrics'` for MOT task on the MOTChallenge benchmarks.
+- `test_evaluator`: the config of testing evaluator, which is similar with `val_evaluator`.
+- `train_cfg`: the config of training loop. For example, `type='EpochBasedTrainLoop'`.
+- `val_cfg`: the config of validation loop. For example, `type='ValLoop'`.
+- `test_cfg`: the config of testing loop. For example, `type='TestLoop'`.
+- `default_hooks`: the config of default hooks, which may include hooks for timer, logger, param_scheduler, checkpoint, sampler_seed, visualization, etc.
+- `vis_backends`: the config of visualization backends, which uses `type='LocalVisBackend'` as default.
+- `visualizer`: the config of visualizer. For example, `type='DetLocalVisualizer'` for VID task, and `type='TrackLocalVisualizer'` for MOT, VIS, SOT, VOS tasks.
+- `param_scheduler`: the config of parameter scheduler, which usually sets the learning rate scheduler.
+- `optim_wrapper`: the config of optimizer wrapper, which contains optimization-related information, for example optimizer, gradient clipping, etc.
+- `load_from`: load models as a pre-trained model from a given path.
+- `resume`: If `True`, resume checkpoints from `load_from`, and the training will be resumed from the epoch when the checkpoint's is saved.
+
+## Modify config through script arguments
+
+When submitting jobs using "tools/train.py" or "tools/test.py",
+you may specify `--cfg-options` to in-place modify the config.
+We present several examples as follows.
+For more details, please refer to [MMEngine](https://github.com/open-mmlab/mmengine/blob/ca282aee9e402104b644494ca491f73d93a9544f/mmengine/config/config.py#L919)
+
+- Update config keys of dict chains.
+
+  The config options can be specified following the order of the dict keys in the original config.
+  For example, `--cfg-options model.detector.backbone.norm_eval=False` changes the all BN modules in model backbones to train mode.
+
+- Update keys inside a list of configs.
```

Comment:
Maybe it would be more comfortable to show it in bold here? Of course, I'm just thinking personally.